mautic/mautic
# FIX: Fixes lookup paths for theme twig files

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

M5 Changed the paths Mautic uses to look for themed twig files:

| | M4 | M5 |
| - | - | - |
| style.html.twig | `THEME/html/MauticFormBundle/Builder/style.html.twig` | `THEME/html/MauticFormBundle/Builder/_style.html.twig` |
| Form fields | `THEME/html/MauticFormBundle/Field/FIELD.html.twig` | `THEME/Field/FIELD.html.twig` |

This might be intentional, or a mistake. If it was a mistake this PR fixes it and ensures backwards compatibility.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Use the paths above for testing
